project/src/main/java/com/google/sps/data/RestaurantScorer.java

```diff
@@ -1,36 +1,60 @@
 package com.google.sps.data;
 
+import com.google.gson.Gson;
+import java.util.HashMap;
+import java.util.Map;
 import org.json.JSONException;
 import org.json.JSONObject;
 
 public final class RestaurantScorer {
+  private static final double EARTH_RADIUS = 6371000; // meters
```

Comment:
Maybe we name this EARTH_RADIUS_METERS so it's clear this is in meters and then maybe we don't need the comment